src/linter.js

```diff
@@ -9,6 +9,7 @@ import { terminalWidth } from 'cli';
 import * as constants from 'const';
 import { ARCH_DEFAULT, ARCH_JETPACK, CHROME_MANIFEST, INSTALL_RDF,
          MANIFEST_JSON } from 'const';
+import { BANNED_LIBRARIES, UNADVISED_LIBRARIES } from 'libraries';
```

Comment:
`UNADVISED` reads weird to me, but that's probably more because it's not commonly used.